packages/as-transformer/src/transformer.ts

```diff
@@ -30,11 +25,14 @@ const protobufTransformerDecorator = 'massaExport';
  */
 export class Transformer extends TransformVisitor {
   visitFunctionDeclaration(node: FunctionDeclaration): FunctionDeclaration {
-    if (hasDecorator(node, protobufTransformerDecorator)) {
-      return transform(node);
-    }
+    let massaNode = MassaFunctionNode.createFromASTNode(node);
 
-    return super.visitFunctionDeclaration(node);
+    for (let transformer of functionTransformers) {
```

Comment:
This is something that we will be able to do if we need to add more transformers. Right now we should focus on finishing the current one.